lib/adBlockRustUtils.js

```diff
@@ -126,11 +126,56 @@ const generateResourcesFile = async (outLocation) => {
   return fs.writeFile(outLocation, await generateResources(), 'utf8')
 }
 
+/**
+ * A list of requests that should not be blocked unless the list has some serious issue.
+ *
+ * Each element is [requestUrl, sourceUrl, requestType].
+ */
+const sampleUnblockedNetworkRequests = [
+  // real resources from a personal website which will never ship ads/trackers
+  ['https://antonok.com', 'https://antonok.com', 'document'],
```

Comment:
Can we move this to a Brave-controlled website?

Reply:
Maybe https://dev-pages.brave.software/filtering/index.html i.e. https://github.com/brave-experiments/qa-test-pages?

Reply:
I figured it's best to have a broad variety of requests here if possible, including multiple domains. I'd be happy to add Brave's QA test pages too, although I believe we _do_ ship a few filters specifically targeting those.

FWIW none of these URLs are actually fetched; they're basically just used as strings for matching purposes.

Reply:
added some requests from there too